scripts/vim-patch.sh

```diff
@@ -85,11 +85,13 @@ clean_files() {
 get_vim_sources() {
   require_executable git
 
+  local update=$1
+
   if [[ ! -d ${VIM_SOURCE_DIR} ]]; then
     echo "Cloning Vim into: ${VIM_SOURCE_DIR}"
     git clone https://github.com/vim/vim.git "${VIM_SOURCE_DIR}"
     cd "${VIM_SOURCE_DIR}"
-  else
+  elif [[ "$update" == 1 ]]; then
```

Comment:
I think "update" string is clearer pattern. And it's used elsewhere .

Reply:
Amended.